rdrf/rdrf/templates/rdrf_cdes/form.html

```diff
@@ -505,6 +506,14 @@
         $("textarea").addClass("form-control");
         $("select").addClass("form-control");
         $("label[for*='-clear']").removeClass();
+
+        var session_refresh_interval = setInterval( function() {
```

Comment:
We should also account for dropped connections. If the request fails or the retries are about to be exhausted, there should be a warning to the user. In the case of a request failure, maybe prompt to check their connection?

Reply:
If the retries are about to be exhausted, maybe prompt them to refresh the page, or click a link to manually reset the retries.

Reply:
Updated.